src/NHibernate/PropertyAccessException.cs

```diff
@@ -82,20 +83,7 @@ protected PropertyAccessException(SerializationInfo info, StreamingContext conte
 		{
 			_propertyName = info.GetString("propertyName");
 			_wasSetter = info.GetBoolean("wasSetter");
-
-			foreach (SerializationEntry entry in info)
-			{
-				switch (entry.Name)
-				{
-					// TODO 6.0: remove "persistentType" deserialization
-					case "persistentType":
-						_persistentType = (System.Type) entry.Value;
-						break;
-					case "_persistentType":
-						_persistentType = (SerializableSystemType) entry.Value;
-						break;
-				}
-			}
```

Comment:
The reason for this was to be highly compatible for deserializing exceptions that had been stored in a database.  I don't actually know if that happens, so if it doesn't happen or is not an issue, then removing this is fine.

Reply:
I do not think we need to guarantee the possibility of deserializing data across minor versions. I would tend to do that only for a patch version.